docs/source/API/containers/DynRankView.rst

```diff
@@ -0,0 +1,452 @@
+
+.. include:: ../../mydefs.rst
+
+``DynRankView``
+===============
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Header File: ``<Kokkos_DynRankView.hpp>``
+
```

Comment:
Please follow a standard for API documentation entries:  function / class / container name, Header File, human language Description, Usage (showing parameters and methods) and Example (copy-paste-able working code).